cdxev/set.py

```diff
@@ -349,12 +350,12 @@ def run(sbom: dict, updates: t.Sequence[dict[str, t.Any]], cfg: SetConfig) -> No
 
     try:
         _validate_update_list(updates, ctx)
-    except AppError:
-        msg = LogMessage(
+    except AppError as e:
+        raise AppError(
```

Comment:
should not this change be applied to [L376](https://github.com/Festo-se/cyclonedx-editor-validator/blob/79b8483d693ad5b10b0c360e81ac7cd7d5530312/cdxev/set.py#L376) too?

Reply:
That's not the same situation. In this line, we previously caught an `AppError` but through away all of the helpful information it contained, instead of exposing it to the user.

In L376 we're building an `AppError` from a `KeyError`. That is not to say that we couldn't maybe use information from the `KeyError` to enrich our own `AppError` but that's a different matter we might or might not look into.